layers/+tools/chrome/packages.el

```diff
@@ -26,3 +27,26 @@
 
 (defun chrome/init-gmail-message-mode ( )
   (use-package gmail-message-mode))
+
+(defun chrome/init-flymd ()
+  (use-package flymd
+    :defer t
+    :init
+    (progn
+      (defun start-browser(browser url)
+        (let ((process-environment (browse-url-process-environment)))
+          (apply 'start-process
+                 "flymd" nil
+                 browser
+                 (list "--new-window" "--allow-file-access-from-files" url))))
+
+      (defun my-flymd-browser-function (url)
+               (cond
+                ((executable-find "chromium") (start-browser (executable-find "chromium") url))
+                ((executable-find "google-chrome") (start-browser (executable-find "goolge-chrome") url))
+                ((executable-find "google-chrome-stable") (start-browser (executable-find "goolge-chrome-stable") url))
+                (chrome-exec-path (start-browser chrome-exec-path url))
```

Comment:
Since you allows users to specify `chrome-exec-path`, I think it would be better to prioritise it over `executable-find` when the value is not nil.